ui/src/components/ripple-out/RippleOutContainer.js

```diff
@@ -0,0 +1,345 @@
+        <Container
+          maxW={['container.xl']}
+          h="100%"
+          textStyle="darker_grotesque"
+        >
+          <Flex flexDir="column" gap={9}>
+            <RippleOutSubHero
+              headerFontSize={headerFontSize}
+              bodyFontSize={bodyFontSize}
+            />
+            <ProgressSection bodyFontSize={bodyFontSize} />
+
+            <Flex flexDir="column" gap={3} ref={stepsRef}>
+              <Text
+                fontSize={headerFontSize}
+                textStyle="darker_grotesque_black"
+                color="#182E57"
+                lineHeight="0.9em"
+              >
+                THE STEPS
+              </Text>
+              <Text fontSize={bodyFontSize}>
+                As you prayerfully consider giving to the campaign, you are not
+                just simply giving a sum to a campaign or facility, but you are
+                giving towards a larger vision so that we can accomplish all
+                that God has called us and see life transformation through the
+                Gospel
+              </Text>
+            </Flex>
+          </Flex>
+        </Container>
+
+        {/* Ripple steps drops image - desktop & mobile */}
+        <Box
+          my={9}
+          h={['60vh', '40vh', '50vh', '85vh']}
+          w="screen"
+          bgImage={`url(${process.env.PUBLIC_URL}/images/ripple-out/ripple-steps.png)`}
+          bgSize="contain"
+          bgRepeat="no-repeat"
+          bgColor="#34486F"
+          display={['none', 'block']}
+        />
+        <Box
+          my={9}
+          h={['40vh', '50vh', '50vh', '50vh']}
+          w="screen"
+          bgImage={`url(${process.env.PUBLIC_URL}/images/ripple-out/ripple-steps-mobile.png)`}
+          bgSize="contain"
+          bgRepeat="no-repeat"
+          bgColor="#34486F"
+          display={['block', 'none']}
+        />
+
+        <RippleOutTakePartSection
+          headerFontSize={headerFontSize}
+          bodyFontSize={bodyFontSize}
+        />
+      </Flex>
+
+      {/* The space section */}
+      <Flex
+        minH="100%"
+        flexDir="column"
+        bgColor="#C9DDED"
+        py={['1.8em', '4em']}
+        color={fontColor}
+        ref={spaceRef}
+      >
+        <RippleOutSpaceSection
+          headerFontSize={headerFontSize}
+          bodyFontSize={bodyFontSize}
+        />
+      </Flex>
+      {/* The support and how to give section */}
+      <Flex
+        minH="100%"
+        flexDir="column"
+        bgColor="#FFFBEF"
+        py={['1.8em', '4em']}
+        color={fontColor}
+        ref={supportRef}
+      >
+        <RippleOutSupportGiveSection
+          headerFontSize={headerFontSize}
+          bodyFontSize={bodyFontSize}
+        />
+      </Flex>
+
+      {/* FAQ section */}
+      <Flex
+        flexDir="column"
+        bgColor="#C9DDED"
+        py={['1.8em', '4em']}
+        color={fontColor}
+      >
+        <RippleOutFaqSection headerFontSize={headerFontSize} />
```

Comment:
move this to the appropriate page according to the design. right now the FAQ appears on every page